containers/signing-service/Dockerfile.j2

```diff
@@ -0,0 +1,19 @@
+FROM debian:{{ debian_version }}
+
+COPY create-user.sh                     /tmp/
+RUN  /tmp/create-user.sh
+
+COPY signing-service/RELEASE-VERSION    /usr/lib/RELEASE-VERSION
+COPY signing-service/requirements.lock  /usr/lib/signing-service/
+COPY testing.pref                       /etc/apt/preferences.d/
+COPY install-dependencies.sh            /tmp/
+COPY install-signing-service.sh         /tmp/
+COPY clean-container.sh                 /tmp/
+COPY middleman_protocol/                /usr/lib/middleman_protocol/
+RUN  /tmp/install-signing-service.sh
+
+USER signing-service
+COPY signing-service/                   /usr/lib/signing-service/
```

Comment:
Please put both `middleman_protocol/` and `signing_service/` as subdirectories in a common directory. For example `/usr/lib/concent/` or `/usr/lib/signing_service/`. Put the virtualenv in that directory too.

We want to have only a single directory in `/usr/lib/` and everything else inside it.

Reply:
Done.